docusaurus/docs-internals/editor-controls/data-controls/toggle.md

```diff
@@ -0,0 +1,177 @@
+| `type`     | `string`                           |    -    | Type should be `"switch"` to use this control                                                                                                                                                                                                                                                                                                     |
+| `label`    | `string`                           |    -    | The label displayed on the left side of the control                                                                                                                                                                                                                                                                                               |
+| `position` | `number`                           |    -    | The position of the control in toolbar                                                                                                                                                                                                                                                                                                            |
+| `devices`  | `"all"`/`"desktop"`/`"responsive"` | `"all"` | Define the devices where the control will be rendered. `"all"` renders the control on all devices. `"desktop"` renders the control only on desktop devices. `"responsive"` renders the control on both tablet and mobile devices.                                                                                                                 |
+| `disabled` | `boolean`                          | `false` | Configure the condition under which the control is disabled or enabled.                                                                                                                                                                                                                                                                           |
+| `choices`  | `Array<Choice>`                    |    -    | A required property that defines the array of choice objects.<br/><br/><b>`Choice: { title?: string, icon: string, value: string \| number }`</b> <br/><br/>`title`: The title of the element, which appears on hover.<br/>`icon`: The icon associated with the element.<br/>`value`: The value returned by the group when an option is selected. |
+
+### Return value
+
+The return value is determined by the configuration of the choices properties.
+
+### Usage
+
+#### Basic example
+
+Standard definition with only the required keys. <br/>
+`Choices` is an array of objects, where each object represents a choice with a specific value and an icon. <br/>This control will be displayed on all devices.
```

Comment:
i think this description should be in the parameters table

you can check example how was explained for slider units: https://github.com/EasyBrizy/Brizy-Local-Editor/pull/217/files#diff-d2894e80027271cdb12bf7f6a39429972d5cd278c805c82fd0ef8b464949961eR34